mirai-console/backend/mirai-console/src/plugin/jvm/JvmPluginClasspath.kt

```diff
@@ -45,9 +45,32 @@ public interface JvmPluginClasspath {
      * [pluginClassLoader] 是否可以通过 [ClassLoader.getResource] 获取 Mirai Console (包括依赖) 的相关资源
      *
      * 默认为 `false`
+     *
+     * @since 2.15.0
      */
+    @SettingProperty("resources.do-resolve-system", defaultValue = "false")
```

Comment:
```suggestion
    @SettingProperty("resources.should.resolve.console.system.resource", defaultValue = "false")
```
property 的意义需要明确, 直接用属性名就可以了